lib/puppet/type/rds_instance.rb

```diff
@@ -0,0 +1,184 @@
+Puppet::Type.newtype(:rds_instance) do
+  @doc = 'type representing an RDS instance'
+
+  ensurable do
+    defaultvalues
+    aliasvalue(:available, :present)
+    newvalue(:destroy) do
```

Comment:
What's the rationale for introducing destroy directly and not just using absent?